diff --git a/UnBox3D/UnBox3D.csproj b/UnBox3D/UnBox3D.csproj
index e773b63..a69b676 100644
--- a/UnBox3D/UnBox3D.csproj
+++ b/UnBox3D/UnBox3D.csproj
@@ -9,6 +9,8 @@
     <UseWindowsForms>true</UseWindowsForms>
     <ApplicationIcon>Assets\Icons\logo.ico</ApplicationIcon>
     <OutputType>WinExe</OutputType>
+    <BaseIntermediateOutputPath>$(TEMP)\UnBox3D\obj\</BaseIntermediateOutputPath>
+    <BaseOutputPath>$(TEMP)\UnBox3D\bin\</BaseOutputPath>
   </PropertyGroup>
 
   <ItemGroup>
